test/e2e/scheduler_predicates.go

```diff
@@ -529,6 +528,14 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 		labelPod, err := c.Pods(ns).Get(labelPodName)
 		framework.ExpectNoError(err)
 		Expect(labelPod.Spec.NodeName).To(Equal(nodeName))
+
+		By("removing the label " + k + " off the node " + nodeName)
```

Comment:
probably also worth doing this in a `defer`, as otherwise it will only run on success.